internal/repository/miggrations/001_init.up.sql

```diff
@@ -1,4 +1,5 @@
 CREATE TABLE IF NOT EXISTS users (
     username VARCHAR(30) PRIMARY KEY,
-    password TEXT NOT NULL
+    password TEXT NOT NULL,
+    created TEXT NOT NULL
```

Comment:
created is text because i can't guarantee equality between time which we can get from golang code and which will be putted into database. In case when we use golang timestamp for salt, and later for password checking timestamp from db i thought to put definetly same value without any oportunities to unpredicted behavior.

Reply:
What do you mean in `equality`?
If you put into database timestamp 19:00:00 12.12.2004, it will be the same as golang time

Reply:
in threads on the internet i saw a few specials for example:
golang timestamp precise is up to nano seconds 10^-9 but postre db is microseconds 10^-6
also seen that one of them stores timezone, bur another not.

but in firstly i thought to use more dynamic salt(date created) instead one secret phrase due to security